src/components/overview/feedback/feedback_mock/feedback.json

```
@@ -0,0 +1,62 @@
+[
+    {
+        "hash": "QmXGuMJBBrJz8iFhvigjsNpRMDiGms2Y18kH5PmYJdUVKS",
+        "feedback": [
+            {
+                "value": "Decent document, but I would put some more effort and time in explaining your metaphores",
+                "pubKey": "bas",
+                "time": 1560416457
+            },
+            {
+                "value": "It's a good start, but it still requires further explanation of the research.",
+                "pubKey": "bart",
+                "time": 1560416500
+            },
+            {
+                "value": "I like your document cover.",
+                "pubKey": "levi",
+                "time": 1560416511
+            },
+            {
+                "value": "test123",
+                "time": 1560419923,
+                "pubKey": "PUBLICKEYOFUSER"
+            }
+        ]
+    },
+    {
+        "hash": "QmbcXF47Xf8aye8eRUoLTBrLJVqeptNjZLDHW3aYt79cjF",
+        "feedback": [
+            {
+                "value": "This is a 1 in a 1000 kind of document, well done.",
+                "pubKey": "bas",
+                "time": 1560416457
+            },
+            {
+                "value": "This should be an example for all other documents",
+                "pubKey": "bart",
+                "time": 1560416500
+            },
+            {
+                "value": "I would put some more work in explaining your reflection, other than that 5/7 document",
+                "pubKey": "levi",
+                "time": 1560416511
+            }
+        ]
+    },
+    {
+        "hash": "QmWScQUnnhcoLrgGU1H7sWv1vNc38djz2kEmQD7dzmjsEC",
+        "feedback": [
+            {
+                "value": "Some would say the document is written unprofessionally, but other are able to see the joke.",
+                "pubKey": "bas",
+                "time": 1560416457
+            },
+            {
+                "value": "This document clearly needs some more work, but I like the basis setup.",
```

Change this sentence.

```suggestion
                "value": "This document clearly needs some more work, but I like the base structure.",
```